src/Microsoft.ML.Analyzer/Microsoft.ML.Analyzer.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
In order to get this analyzer in our `Microsoft.ML` NuGet package, we will have to do some build authoring. Today our infrastructure only supports putting things in the `build`, `lib` and `runtimes` folders. It shouldn't be much work, but someone will have to do it.

Let me know if you want me to pick the NuGet authoring up, or if you want to handle it yourself (I can give you some tips if necessary).